tests/lib/rules/max-len.js

```diff
@@ -76,6 +76,11 @@ ruleTester.run("max-len", rule, {
             options: [80, { tabWidth: 4, comments: 30 }]
         }, {
             code:
+                "// I like longer comments and shorter code\n" +
+                "function see() { odd(eh()) }",
+            options: [30, { tabWidth: 4, comments: 80 }]
```

Comment:
@ljharb I think this test satisfies your request for one where comments have a _shorter_ length than code, and the comment length falls in between the comment and code lengths (comment length is 42)